app/controllers/works_controller.rb

```diff
@@ -2,6 +2,7 @@ class WorksController < ApplicationController
   # We should always be able to tell what category
   # of work we're dealing with
   before_action :category_from_work, except: [:root, :index, :new, :create]
+  before_action :require_login, except: [:root]
 
   def root
```

Comment:
You've also added `require_login` as a filter on the `ApplicationController`, which means it will happen before every single controller action. Your line 5 here adds it again for everything but `root`, but doesn't take it away for `root`. Instead it should be a `skip_before_action` for only `:root`.

This is also why all your `WorksController` tests were breaking.

This was a tricky one - it took me a while to figure out what was going on 🙂